browse/domain/metadata.py

```diff
@@ -117,7 +117,7 @@ class DocMetadata:
     categories: str
```

Comment:
Should this (in DocMetadata) need to be Option[str] now to accommodate the possible `None` assignment that is added? Not sure how it typechecks, but I haven't tried running it locally yet.

Reply:
It seems to typecheck fine but I think `Optional[str]` would be clearer so I'll try that.